editor/plugins/tiles/tile_data_editors.cpp

```diff
@@ -859,6 +859,11 @@ void GenericTilePolygonEditor::_notification(int p_what) {
 				button_expand->set_pressed_no_signal(false);
 			}
 		} break;
+
+		case NOTIFICATION_READY: {
+			get_parent()->connect("tree_exited", callable_mp(TileSetEditor::get_singleton(), &TileSetEditor::remove_expanded_editor));
```

Comment:
Does it really need to be connected to the parent? Wouldn't it work to simply put it in a `NOTIFICATION_EXIT_TREE`, as I assume GenericTilePolygonEditor exits the tree too when its parent does ?

Reply:
Expanded editor has different parent and does not exit.